### ISSUE TITLE
[Bug] tcprewrite fails with "From tcpedit.c:tcpedit_packet() line 192: Packet length 98 is to short to contain an IPv6 header for DLT 0x0001"

### ISSUE THREAD

**Issue by yp-isi** [appneta/tcpreplay#611]
**Describe the bug**
TCP rewrite fails to parse input pcap file, exits with the error:
```
Fatal Error: Error rewriting packets: From tcpedit.c:tcpedit_packet() line 192:
Packet length 98 is to short to contain an IPv6 header for DLT 0x0001
```

**To Reproduce**
1. Run `tcprewrite` as follows:
```
tcprewrite -vv -i afail1.pcap -o /dev/null --pnat="9.9.0.0/16:1.1.0.0/16"
```
2. Observe the output:
```
reading from file -, link-type EN10MB (Ethernet)
17:11:17.1592871077 IP6 4950:3aa3:964b:a356:60a3:7dea:a836:3200 > 4efa:695:ee19:12d9:4906:6f8b:8bbb:6bd6: frag (0|36) 63927 > domain: 0 [0q] (28)

Fatal Error: Error rewriting packets: From tcpedit.c:tcpedit_packet() line 192:
Packet length 98 is to short to contain an IPv6 header for DLT 0x0001
```

**Expected behavior**
Not to fail.  Even if the packet is not parse-able, it should either (a) drop it, (b) overwrite IPs, but not checksum, (c) leave such packets as is.

**Screenshots**
N/A

**System (please complete the following information):**
 - OS: linux
 - OS fedora 31
 - Tcpreplay Version
```
tcpreplay version: 4.3.2 (build git:v4.3.2)
Copyright 2013-2018 by Fred Klassen <tcpreplay at appneta dot com> - AppNeta
Copyright 2000-2012 by Aaron Turner <aturner at synfin dot net>
The entire Tcpreplay Suite is licensed under the GPLv3
Cache file supported: 04
Compiled against libdnet: 1.12
Compiled against libpcap: 1.9.1
64 bit packet counters: enabled
Verbose printing via tcpdump: enabled
Packet editing: disabled
Fragroute engine: enabled
Injection method: PF_PACKET send()
Not compiled with netmap
```

**Additional context**
Failed pcap attached.
[afail1.pcap.gz](https://github.com/appneta/tcpreplay/files/4834199/afail1.pcap.gz)

**Comment by fklassen** [comment]
I appears that the packet includes an IPv6 fragmentation header. Those are disabled by default, so it is extremely rare to find a capture with one in them.

I'm changing this to an "enhancement" since this packet type has never been supported.

**Comment by fklassen** [comment]
It appears to be working in 4.4.0-beta2. It was fixed with PR #677 which did some rewrites in general packet handling.

Marked as fixed.
